### PR TITLE
Sphinx documentation

### PR DESCRIPTION
**Pull request by mrkickling**

### CODE REVIEW

File: maltoolbox/language/classes_factory.py [hunk 1]
```diff
@@ -25,7 +26,7 @@ def __init__(self, lang_graph: LanguageGraph):
 
     def _generate_assets(self) -> None:
         """
-        Generate JSON Schema for the assets in the language specification.
+        Generate JSON Schema for asset types in language specification.
```
Comment (andrewbwm) [inline maltoolbox/language/classes_factory.py:29]: The first 'the' being removed makes sense, the second one does not. :)

File: maltoolbox/language/classes_factory.py [hunk 2]
```diff
@@ -67,7 +68,7 @@ def _generate_assets(self) -> None:
 
     def _generate_associations(self) -> None:
         """
-        Generate JSON Schema for the associations in the language specification.
+        Generate JSON Schema for association types in language specification.
```
Comment (andrewbwm) [inline maltoolbox/language/classes_factory.py:71]: Same as above.

File: maltoolbox/wrappers.py [hunk 3]
```diff
@@ -24,11 +24,17 @@ def create_attack_graph(
...
+    ----------
+    :param lang_file: path to language file (.mar or .mal)
+    :param model_file: path to model file (yaml or json)
+    :param attach_attackers: whether to run attach_attackers or not
+    :param calc_viability_and_necessity: run apriori calculations or not
+
+    Returns
+    -------
+    AttackGraph
+        AttackGraph generated using model given
```
Comment (andrewbwm) [inline maltoolbox/wrappers.py]: Why did the format change here and only here?

File: maltoolbox/language/classes_factory.py (hunk 2 above)
Comment (mrkickling) [reply maltoolbox/language/classes_factory.py]: fixed

File: maltoolbox/language/classes_factory.py (hunk 1 above)
Comment (mrkickling) [reply maltoolbox/language/classes_factory.py]: fixed

**Review by andrewbwm — APPROVED** [review]

**Pull request merged**